CMPUT301Project/app/src/main/java/com/example/GroupProject/MainActivity.java

```diff
@@ -47,4 +62,8 @@ protected void onCreate(Bundle savedInstanceState) {
         // Access a Cloud Firestore instance from your Activity
         db = FirebaseFirestore.getInstance();
     }
+
+    public static FirebaseFirestore getFirestoreInstance(){
```

Comment:
Seems redundant, just call FirebaseFirestore.getinstance directly. Also doesn't look like you're using this function anywhere.